content/panorama/scripts/custom_game/hero_selection.js

```diff
@@ -113,7 +115,8 @@ function onPlayerStatChange (table, key, data) {
   var newimage = null;
   if (key === 'herolist' && data != null) {
     currentMap = data.gametype;
```

Comment:
You assign currentMap above, it shouldn't be reassigned. If `Game.GetMapInfo().map_display_name` works then lets use that, if it doesn't then this can stay.